x-pack/plugins/index_lifecycle_management/public/application/services/policies/policy_serialization.test.ts

```diff
@@ -436,6 +436,48 @@ describe('Policy serialization', () => {
     });
   });
 
+  test('serialize a policy using "best_compression" codec for forcemerge', () => {
+    expect(
+      serializePolicy({
```

Comment:
Thanks for adding this test case! It would be worth testing the case when `delete` action has a value set in it - WDYT?

Reply:
thanks for your comment, @jloleysens ! I added a test to make sure that an existing `delete` action is not overwritten.